crypto/src/fields.rs

```diff
@@ -56,18 +38,18 @@ pub fn scalar_to_bigdecimal(a: &Scalar) -> BigDecimal {
     BigDecimal::from(bigint)
 }
 
-/// Converts a dalek scalar to an arkworks ff element
-pub fn scalar_to_prime_field(a: &Scalar) -> DalekRistrettoField {
-    Fp256::from(scalar_to_biguint(a))
+pub fn scalar_to_u64(a: &Scalar) -> u64 {
```

Comment:
nit:
could we call this `reduce_scalar_to_u64`, or leave a doc comment indicating that this takes the 8 least significant bytes of the scalar?

Reply:
Added a doc comment, want to keep the method signature the same because it's used a bunch of places. Also added a lint denial for missing docs on this library so that we don't have any more methods without docs.